themes/blue-owl.omp.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "final_space": false,
+  "osc99": true,
+  "console_title": true,
+  "console_title_style": "template",
+  "console_title_template": "{{if .Root}} ⚡ {{end}}{{.Folder | replace \"~\" \"🏚\" }} @ {{.Host}}",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "root",
+          "style": "plain",
+          "foreground": "#FFEB3B",
+          "background": "#a313a8",
+          "properties": {
+            "root_icon": "⚡"
+          }
+        },
+        {
+          "type": "os",
+          "style": "plain",
+          "foreground": "#ffffff",
+          "background": "transparent"
+        },
+        {
+          "type": "path",
+          "style": "diamond",
+          "leading_diamond": "<transparent,#01579B>\uE0B0</>",
+          "trailing_diamond": "\uE0B0",
+          "foreground": "#ffffff",
+          "background": "#01579B",
+          "properties": {
+            "home_icon": "\uF7DB",
+            "folder_icon": "",
```

Comment:
Missed a few 😉

Reply:
Thanks, fixed the folder_icon to \u representation.